bin/utils.sh

```diff
@@ -492,17 +492,17 @@ parse_args() {
 
 # create or update the insecure secrets by setting the 3 required fields in Consul
 set_insecure_secret() {
-    put_insecure_secrets_field "${SECRET_PATH}/Path"                "${SECRET_PATH}"
-    put_insecure_secrets_field "${SECRET_PATH}/Secrets/username"    "${SECRET_USERNAME}"
-    put_insecure_secrets_field "${SECRET_PATH}/Secrets/password"    "${SECRET_PASSWORD}"
-    put_insecure_secrets_field "${SECRET_PATH}/Secrets/mode"        "${AUTH_MODE}"
+    put_insecure_secrets_field "${SECRET_NAME}/SecretName"                "${SECRET_NAME}"
+    put_insecure_secrets_field "${SECRET_NAME}/SecretData/username"    "${SECRET_USERNAME}"
+    put_insecure_secrets_field "${SECRET_NAME}/SecretData/password"    "${SECRET_PASSWORD}"
+    put_insecure_secrets_field "${SECRET_NAME}/SecretData/mode"        "${AUTH_MODE}"
 }
 
 # set the secure secrets by posting to the device service's secret endpoint
 set_secure_secret() {
     local payload="{
     \"apiVersion\":\"v2\",
-    \"path\": \"${SECRET_PATH}\",
+    \"path\": \"${SECRET_NAME}\",
```

Comment:
@vli11 does this need to be changed from `"path"`?

Reply:
Yes.